pkg/dogstatsd/replay/writer.go

```diff
@@ -261,6 +261,9 @@ process:
 			}
 		case <-shutdown:
 			log.Debug("Capture shutting down")
+			tc.Lock()
+			tc.ongoing = false
```

Comment:
If we set `ongoing` to false here, before we completely finished the capture, would it potentially cause problems if another `capture start` command was issued between here and the end of this routine?

Reply:
Good point, will have to think this over again. The other changes I made was just cleaning up access (making things a bit more private when no obvious reason for them to be public). Will clean up the PR when I tackle this issue 👍🏽